### ISSUE TITLE
Keyboard navigation traversing hidden nodes

### ISSUE THREAD

**Issue by pixelchutes** [500tech/angular2-tree-component#67]
Perhaps this is a feature, or supported for some other use case, but I noticed when a node is hidden _(either by design, or as a result of `filterNodes`)_, that when you navigate using the keyboard you may get unexpected results (visual selection indicator not lining up with actual key presses).

For example, if a hidden node lives above your current selection, you have to press <kbd>Up</kbd> _twice_ in order to select the node actually displayed immediately above. Now, if that hidden node has _children_, you have to press it that many more times as well.

```JavaScript
keys: {
    [KEYS.DOWN]: TREE_ACTIONS.NEXT_NODE,
    [KEYS.UP]: TREE_ACTIONS.PREVIOUS_NODE
}
```

**Comment by adamkleingit** [comment]
This is not by design, I'll solve it in the next version

**Comment by adamkleingit** [comment]
Fixed in 1.3.3
